CodeFormatter/formatoptions.h

```diff
@@ -112,6 +112,16 @@ enum PHPFixserFormatterStyle {
     kPcfConcatSpaceOne = (1 << 19),
     kPcfEmptyReturnStrip = (1 << 20),
     kPcfEmptyReturnKeep = (1 << 21),
+    kPcfCombineConsecutiveUnsets = (1 << 22),
+    kPcfLinebreakAfterOpeningTag = (1 << 23),
+    kPcfMbStrFunctions = (1 << 24),
+    kPcfNoBlankLinesBeforeNamespace = (1 << 25),
+    kPcfNoMultilineWhitespaceBeforeSemicolons = (1 << 26),
+    kPcfNoNullPropertyInitialization = (1 << 27),
+    kPcfNoPhp4Constructor = (1 << 28),
+    kPcfNoShortEchoTag = (1 << 29),
+    kPcfNoUnreachableDefaultArgumentValue = (1 << 30),
+    kPcfNoUselessElse = (1 << 31),
```

Comment:
split them.
You can use up 64 if you use macros instead of `enum`, like this:

```C++
#define kOne (1ULL << 0)
#define kTwo (1ULL << 1)
#define kThree (1ULL << 2)
#define kLast (1ULL << 33)
```